extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/permission/ModeledObjectPermissionService.java

```diff
@@ -47,14 +49,14 @@ protected ObjectPermission getPermissionInstance(ObjectPermissionModel model) {
     }
 
     @Override
-    protected ObjectPermissionModel getModelInstance(ModeledUser targetUser,
+    protected ObjectPermissionModel getModelInstance(
+            ModeledPermissions<? extends EntityModel> targetEntity,
             ObjectPermission permission) {
 
         ObjectPermissionModel model = new ObjectPermissionModel();
```

Comment:
Only that there is no constructor of `ObjectPermissionModel` which populates those fields. We could write one.

Reply:
I don't know that it needs to hold up this particular PR, and I don't know how valuable it will be.  Seems like I ran across several instances in various places that had this sort of empty constructor plus several `set` methods, so probably worth a separate cleanup effort.